docs/changelog.md

```diff
@@ -4,6 +4,15 @@ tocMaxDepth: 2
 
 # OpenAPI CLI changelog
 
+## 1.0.0-beta.88 (2022-03-16)
+
+### Fixes
+
+- Fixed an issue with the `lint` command highlighting entire file when `servers` are missing in OAS3 (now it highlights `openapi` field indicating incorrect OpenAPI definition).
+- Fixed an issue with the `lint` command highlighting all the parent value when one of the child fields has empty value instead of field itself.
```

Comment:
```suggestion
- Fixed an issue with the `lint` command highlighting the entire file when `servers` are missing in OAS3. Now it highlights only the `openapi` field, indicating an incorrect OpenAPI definition.
- Fixed an issue with the `lint` command highlighting all parent values when one of the child fields has an empty value instead of highlighting the field itself.
```